tema5/src/main/java/ro/sci/carrental/Main.java

```diff
@@ -26,7 +28,7 @@ public static void main(String[] args) throws InvalidCurrencyException {
 
         /* reads from a text file, and the text is transformed to a Car object
         * */
-        final List<Car> carList = new ArrayList<Car>();
+        /* final List<Car> carList = new ArrayList<Car>();
         File file = new File("tema5/src/main/java/cars.txt");
         EntityReader entityReader = new EntityReader();
         List<String> lines = entityReader.readLines(file);
```

Comment:
fain ar fi ca aci sa te folosesti de CarService si sa adaugi toate masinile astea in DB ;)